README.md

```diff
@@ -2,12 +2,7 @@
 Working draft of the proposed RISC-V V vector extension.
 
 [Version 1.0](https://github.com/riscv/riscv-v-spec/releases/tag/v1.0)
-has been frozen and at this time is undergoing public review.
-Version 1.0 is considered stable enough to begin developing
-toolchains, functional simulators, and implementations, including in
-upstream software projects, and is not expected to have incompatible
-changes except if serious issues are discovered during
-ratification. Once ratified, the spec will be given version 2.0.
+has been frozen and ratified.
```

Comment:
Add words similar to "This repository now represents work towards a future version 2.0"